### ISSUE TITLE
Qubes Video Companion can't be used to access both screenshare and webcam

### ISSUE THREAD

**Issue by ben-grande** [QubesOS/qubes-issues#9272]
[How to file a helpful issue](https://www.qubes-os.org/doc/issue-tracking/)

### Qubes OS release

R4.2, QVC 1.0.7-1.

### Brief summary

I want to do a call in Signal Desktop in qube `signal`, access webcam from my USB qube `disp-sys-usb` and share the screen of my browser `disp9999`.

The current Video Companion does not allow to use both webcam and screenshare at the same time as it put's a lock.

### Steps to reproduce

On the `signal` (client/receiver) qube:
```sh
qubes-video-companion screenshare
```
Target `disp999`.

And on another shell:
```sh
qubes-video-companion webcam
```

Exits with error:

> Qubes Video Companion is already running! Please stop the previous session before starting a new one.

### Expected behavior

Able to call qubes-video-companion to share screen and webcam at the same time.

### Actual behavior

Can't do both at the same time from a single client.

**Comment by marmarek** [comment]
Duplicate of https://github.com/QubesOS/qubes-issues/issues/9625

**Comment by marmarek** [comment]
Closing the earlier one, as the other one has more details.
